index.js

```diff
@@ -23,7 +34,183 @@ function closeNav() {
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://fithamlak.github.io/',
+    source: 'https://github.com/fithamlak/My-portfolio',
+  },
+  {
+    id: 1,
+    title: 'Multi-Post Stories',
+    skill: ['Canopy', 'Back End Dev', '2015'],
+    description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    ImageForMobile: './images/second-work.svg',
+    ImageForDesktop: './images/work2-desktop-view.svg',
+    accomplishmentDetailModel: './images/accomplishment-details-model.svg',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://fithamlak.github.io/',
+    source: 'https://github.com/fithamlak/My-portfolio',
+  },
+  {
+    id: 2,
+    title: 'Tonic',
+    skill: ['Canopy', 'Back End Dev', '2015'],
+    description:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    ImageForMobile: './images/third-work.svg',
+    ImageForDesktop: './images/work3-desktop-view.svg',
+    accomplishmentDetailModel: './images/accomplishment-details-model.svg',
+    alt: 'work_tonic_daily_personalized_reads',
+    live: 'https://fithamlak.github.io/',
+    source: 'https://github.com/fithamlak/My-portfolio',
+  },
+  {
+    id: 3,
+    title: 'Multi-Post Stories',
+    skill: ['Canopy', 'Back End Dev', '2015'],
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    ImageForMobile: './images/fourth-work.svg',
+    ImageForDesktop: './images/work4-desktop-view.svg',
+    accomplishmentDetailModel: './images/accomplishment-details-model.svg',
+    alt: 'work_multi-post_stories_daily_personalized_reads',
+    live: 'https://fithamlak.github.io/',
+    source: 'https://github.com/fithamlak/My-portfolio',
+  },
+];
+
+// function to display details popup windo when seeProjrct buttons of each accomplishment clicked
+function seeProject(k) {
```

Comment:
- [ ] [OPTIONAL] Kindly make sure the right image is displayed on the popup, as for now only one image is displayed on all the popups

Reply:
Yes,  I intentionally display one image  for desktop version, and its own different image for the mobile version